gcn/notices/icecube/LVK_nu_track_search.schema.json

```
@@ -0,0 +1,91 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/gcn/notices/icecube/LVK_nu_track_search.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "IceCube LVK Alert Nu Track Search",
+  "description": "IceCube LVK Coincident Neutrino Track Search",
+  "anyOf": [
+    {
+      "$ref": "/schema/gcn/notices/core/Alert.schema.json",
+      "description": "Alert information (alert_datetime and alert_type) from LVK alert notices"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/FollowUp.schema.json",
+      "description": "ID (ref_ID) of LVK alert notice"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/AdditionalInfo.schema.json"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/Reporter.schema.json",
+      "description": "Record/Sequence number of LVK alerts"
+    },
+    {
+      "$ref": "/schema/gcn/notices/core/DateTime.schema.json"
+    }
+  ],
+  "properties": {
+    "pval_generic": {
+      "description": "P-value from generic transient search; consistency with background expectations [0-1]",
+      "type": "number"
+    },
+    "pval_bayesian": {
+      "description": "P-value from LLAMA bayesian search; consistency with background expectations [0-1]",
```

"Bayesian" is capitalized:

```suggestion
      "description": "P-value from LLAMA Bayesian search; consistency with background expectations [0-1]",
```